airlock_processor/host.json

```diff
@@ -8,8 +8,8 @@
       }
     }
   },
-  "extensionBundle": {
+"extensionBundle": {
     "id": "Microsoft.Azure.Functions.ExtensionBundle",
-    "version": "[4.0.0, 5.0.0)"
+    "version": "[4.*, 5.0.0)"
```

Comment:
So you have confirmed that the current versions we use don't support this?

Reply:
Not yet, planning to do confirm first thing this morning.

Reply:
confirmed, it works now with 4.0.0